presto-memory/src/main/java/com/facebook/presto/plugin/memory/MemoryPageSinkProvider.java

```diff
@@ -65,6 +65,7 @@ public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHa
         checkState(memoryInsertTableHandle.getActiveTableIds().contains(tableId));
 
         pagesStore.cleanUp(memoryInsertTableHandle.getActiveTableIds());
+        pagesStore.initialize(tableId);
```

Comment:
Oh, I'm sorry... I might have tested with other branch and misunderstood all test succeeded.
Exactly this commit introduce a bug.